cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```
@@ -326,3 +326,27 @@ spec:
       for: 0s
       labels:
         severity: critical
+  - name: pool-quota.rules
+    rules:
+    - alert: PoolQuotaBytesNearExhaustion
+      annotations:
+        description: Storage pool {{ $labels.name }} quota usage has crossed 70%.
+        message: Storage pool quota(bytes) is near exhaustion.
+        severity_level: warning
+        storage_type: ceph
+      expr: |
+        (ceph_pool_stored_raw * on (pool_id) group_left(name)ceph_pool_metadata) / ((ceph_pool_quota_bytes * on (pool_id) group_left(name)ceph_pool_metadata) > 0) > 0.70
+      for: 1m
+      labels:
+        severity: warning
+    - alert: PoolQuotaBytesCriticallyExhausted
```

Considering the support of other Storage Providers, we can add Ceph to the name of the alert.
```suggestion
    - alert: CephPoolQuotaBytesCriticallyExhausted
```